src/app/core/services/connector-limits.service.ts

```diff
@@ -0,0 +1,49 @@
+import {Injectable} from '@angular/core';
+import {Observable, of} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {ConnectorLimits} from '@sovity.de/edc-client';
+import {ActiveFeatureSet} from '../config/active-feature-set';
+import {EdcApiService} from './api/edc-api.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ConnectorLimitsService {
+  constructor(
+    private edcApiService: EdcApiService,
+    private activeFeatureSet: ActiveFeatureSet,
+  ) {}
+
+  fetchLimits(): Observable<ConnectorLimits> {
+    return this.edcApiService.getEnterpriseEditionConnectorLimits();
+  }
+
+  limitExceeded(): Observable<boolean> {
```

Comment:
you should maybe enhance the name a bit, as the service is called very generically "ConnectorLimitsService" and call it "isConsumingAgreementLimitExceeded()"